em-client/src/models.rs

```diff
@@ -1051,14 +1051,20 @@ pub struct ApplicationConfigWorkflow {
     #[serde(rename = "port_map")]
     pub port_map: SortedHashMap<String, SortedHashMap<String, models::ApplicationConfigPort>>,
 
+    #[serde(skip_serializing_if = "Option::is_none")]
```

Comment:
Can we use the deny_unknown_fields attribute for `ApplicationConfigWorkflow` struct? This will ensure strict serialize/deserialize and easier to track changes in the data structure from em side.